packages/sdk/src/swap/consts.ts

```diff
@@ -1,4 +1,7 @@
 import { ChainflipNetworks } from '@/shared/enums';
+import { version } from '../../package.json';
+
+export const CF_SDK_VERSION_HEADERS = { 'Cf-Sdk-Version': version };
```

Comment:
```suggestion
export const CF_SDK_VERSION_HEADERS = { 'X-Cf-Sdk-Version': version };
```

Reply:
i asked chatgpt how to name the header yesterday lol, it told me that the `X-` prefix is not a recommendation anymore. but no strong opinion, im fine with adding it if you want